samples/rum-schema/rum/extended_error.json

```diff
@@ -39,16 +39,16 @@
       "carrier_name": "AT&T"
     }
   },
+  "action": {
+    "id": "ae3a5d82-cdd1-468d-9bc9-3aa9e54d953c"
+  },
+  "_dd": {
+    "format_version": 2,
   "display": {
     "viewport": {
       "width": 1920,
       "height": 1080
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏some weird indentation